docs/architecture/overview.md

```diff
@@ -208,10 +208,15 @@ comfortable that the new implementation is stable, we could cut users over to
 the new implementation by simply changing the definition of the user-facing
 view.
 
-## Known Issues
-
-- Hard limit of 10,000 columns per table in BigQuery
-- Max of 100,000 streaming inserts per second per BigQuery table
+## Limits
+
+- The maximum `Content-Length` accepted at the edge is 1 MB; larger payloads
```

Comment:
It's possibly worth referencing the HTTP edge spec and that the "dropped" mechanism is a 413.